rules/windows/defense_evasion_scheduledjobs_at_protocol_enabled.toml

```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/11/23"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/23"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies attempts to enable the Windows Scheduled Jobs AT protocol. Attackers may use this method to move laterally or
+persist locally via the AT Jobs Protocol, which has been deprecated since Windows 8 and Windows Server 2012.
```

Comment:
```suggestion
Identifies attempts to enable the Windows scheduled tasks AT command via the registry. Attackers may use this method to move laterally or
persist locally. The AT command has been deprecated since Windows 8 and Windows Server 2012, but still exists for backwards compatibility.
```